src/content/docs/style-guide/writing-docs/writer-workflow/process-changes.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Proposing process or workflow changes
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Use this document to learn how to propose changes to the process or workflow in a collaborative manner.
+
```

Comment:
Is this doc for any team's use or primarily for our team? Also, we used to have a fairly strong mandate to avoid "ing" verb form for doc titles, primarily in the hopes of better SEO. (For example, readers would be more likely to search for "delete account" rather than "deleting an account.") Not sure if that's still a requirement. Style guide has changed a lot, so you may want to check on that.